pkg/controller/admissionchecks/multikueue/multikueuecluster_test.go

```diff
@@ -376,7 +377,8 @@ func TestUpdateConfig(t *testing.T) {
 			builder = builder.WithStatusSubresource(slices.Map(tc.clusters, func(c *kueue.MultiKueueCluster) client.Object { return c })...)
 			c := builder.Build()
 
-			adapters, _ := jobframework.GetMultiKueueAdapters()
+			enabledIntegrations := sets.New([]string{"batch/job"}...)
+			adapters, _ := jobframework.GetMultiKueueAdapters(enabledIntegrations)
```

Comment:
```suggestion
			adapters, _ := jobframework.GetMultiKueueAdapters(sets.New("batch/job"))
```
The same goes in a couple of other places.